### DISCUSSION TITLE
Failed to start 2

### DISCUSSION THREAD

**Discussion by Cristobal-84** [SafeExamBrowser/seb-win-refactoring#204] [Q&A]
Please advise on the failed to start issued as attached. I have try many way even install and reinstall the 3.3.0.354 version.
I use SEB with Moodle 3.9
<img width="459" alt="SEB12" src="https://user-images.githubusercontent.com/76251099/134208702-6ba66921-fc56-4e85-9b35-2a80568bfadf.png">
Erreur de configuration -> Configuration error
La ressource de configuration ... n'est pas supportée -> The configuration resource ... is not supported

[2021-09-21_11h44m14s_Runtime.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/7204735/2021-09-21_11h44m14s_Runtime.log)

**Comment by dbuechel** [comment]
There appears to be an issue on the server you're trying to download the configuration from. I suggest you contact the technical support of your institution.
